docs/compatibility.md

```diff
@@ -11,9 +11,9 @@ Below are the [semver](http://semver.org/) ranges representing which versions of
 * MongoDB Server 3.2.x: mongoose `^4.3.0` or `5.x`
 * MongoDB Server 3.4.x: mongoose `^4.7.3` or `5.x`
 * MongoDB Server 3.6.x: mongoose `5.x`
-* MongoDB Server 4.0.x: mongoose `^5.2.0`
-* MongoDB Server 4.2.x: mongoose `^5.7.0`
-* MongoDB Server 4.4.x: mongoose `^5.10.0`
+* MongoDB Server 4.0.x: mongoose `^5.2.0` or `6.x`
```

Comment:
AFAIK, we don't drop support for older MongoDB versions as we upgrade versions, so I assume it's safe to change this with:
```suggestion
* MongoDB Server 4.0.x: mongoose >= `^5.2.0`
```

Reply:
I think that `^5.2.0` or `6.x` is less brittle, because otherwise what happens with Mongoose 7?